source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -3377,11 +3377,12 @@ function raiseInlineCardExpandedEvent(action: ShowCardAction, isExpanded: boolea
 
 function raiseElementVisibilityChangedEvent(element: CardElement,
                                             shouldUpdateLayout: boolean = true) {
+    let card = element.getRootElement() as AdaptiveCard;
```

Comment:
Technically, the root element does not have to be a card. Your code shouldn't make this assumption.